azext_iot/common/_azure.py

```diff
@@ -55,9 +55,10 @@ def get_iot_central_tokens(cmd, app_id, token, central_dns_suffix):
         ]
         token = "Bearer {}".format(aad_token)
 
-    url = "https://{}.{}/system/iothubs/generateSasTokens".format(
+    url = "https://{}.{}/system/iothubs/generateEventSasTokens".format(
```

Comment:
Is there a possible issue with API versioning (ex: the old API expects the url to be generateSasTokens and with this change, you can no longer poke the old version - can you still access anything that was created in the old version with the new version and vice versa?)

Reply:
the old API is still active and we have a dismission process going on. I had an offline conversation with @digimaun. IoT Central team will take care of deprecation.

Reply:
the old API was returning more information than needed by CLI and exposed an unwanted access token.